### PR TITLE
Create zffu.json

### PR DESCRIPTION
**Pull request by Zffu**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

Here is my website hosted on Github Pages https://zffu.github.io

### CODE REVIEW

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/zffu.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "zffu.is-a.dev",
+  "owner": {
+    "username": "Zffu",
+    "email": "zffumc@gmail.com",
+    "discord": "Zffu"
+  },
+  "record": {
+    "URL": "zffumc.github.io"
```
[the remark this answers is not in the document]
Comment (YouFoundAlpha) [reply domains/zffu.json]: User has updated the URL record and it is reachable

**Review by YouFoundAlpha — APPROVED** [review]
Looks good but found one issue https://github.com/is-a-dev/register/pull/13013#pullrequestreview-1962466529

**Review by YouFoundAlpha — CHANGES_REQUESTED** [review]
Please add https:// to the beginning of the record to make it redirect to zffu.github.io

File: domains/zffu.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "zffu.is-a.dev",
+  "owner": {
+    "username": "Zffu",
+    "email": "zffumc@gmail.com",
+    "discord": "Zffu"
+  },
+  "record": {
+    "URL": "zffu.github.io"
```
Comment (Stef-00012) [inline domains/zffu.json:9]: URL record must have a protocol (`https://`)

**Review by Stef-DP — APPROVED** [review]

File: domains/zffu.json (hunk 2 above)
Comment (Stef-00012) [inline domains/zffu.json:9]: URL record must have a protocol (https://)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/zffu.json (hunk 2 above)
Comment (Stef-00012) [reply domains/zffu.json:9]: (approved for error before)

Comment (21Z) [inline domains/zffu.json:9]:
```suggestion
    "CNAME": "zffu.github.io"
```

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/zffu.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "zffu.is-a.dev",
+  "owner": {
+    "username": "Zffu",
+    "email": "zffumc@gmail.com",
+    "discord": "Zffu"
+  },
+  "record": {
+    "CNAME": "https://zffu.github.io"
```
Comment (Stef-00012) [inline domains/zffu.json:9]: CNAME can not contain `https://`

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Review by Stef-DP — APPROVED** [review]
seems good to me

**Pull request merged**